contracts/gateway.yaml

```diff
@@ -1822,8 +1823,122 @@ paths:
                 $ref: '#/components/schemas/ErrorResponse'
             application/xml:
               schema:
-                $ref: '#/components/schemas/ErrorResponse'                
-  
+                $ref: '#/components/schemas/ErrorResponse'
+  /v0.5/patient/share:
+    post:
+      tags:
+        - patient
+        - hip facing
+      parameters:
+        - $ref: "#/components/parameters/authorization"
+        - $ref: "#/components/parameters/X-HIP-ID"
+      summary: Patient details on Share by user
+      description: >
+        API to submit the patient details on QR code scan.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PatientShareRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/PatientShareRequest'
+      responses:
+        '202':
```

Comment:
Since the user waits for the whole process to be finished a `200` would be better than `202`. If we were making this call async where user gets a notification later a `202` would have been appropriate.

Reply:
Its already done